iqrm/core.py

```diff
@@ -64,6 +66,7 @@ def iqrm_mask(x, radius=5, threshold=3.0):
         and the values are the set of array indices that received a vote from i.
     """
     x = np.asarray(x)
+    x[ignorechans] = np.nan
```

Comment:
Mutating user input is frowned upon in general, can you change the first line to `x = np.asarray(x, copy=True)` ?

Reply:
I changed it to 
`x = np.array(x)`
which makes a copy by default.
I apparently had an older version of numpy in my iqrm environment and so `copy=True` threw an error. I'm happy to change it to `copy=True` if that's your preference, but this way you don't have to add in a numpy version requirement

Reply:
That's good, thanks